### ISSUE TITLE
Quote

### ISSUE THREAD

**Issue by rowanc1** [iooxa/ink-article#4]

**Comment by rowanc1** [comment]
![image](https://user-images.githubusercontent.com/913249/79406434-4a5fe380-7f54-11ea-8bde-1bdf3bf75a59.png)

I have gone with a pure HTML approach. Means a bit more structuring and remembering. But this is what HTML is for I suppose. :)

```html
<blockquote>
  <p>I would have written a shorter letter, but I did not have the time.</p>
  <footer>
    <cite>Blaise Pascal</cite>
    <time datetime="1656-12-04">December 4, 1656</time>
  </footer>
</blockquote>
```

**Comment by rowanc1** [comment]
Inspiration:
https://codepen.io/wetandsalty/pen/zQxqWq

**Comment by rowanc1** [comment]
Closed in #8
